static/js/gtm.js

```diff
@@ -8,38 +8,20 @@ window.dataLayer.push({
   event: 'gtm.js'
 });
 
-function gtag() { dataLayer.push(arguments); }
 
 // Setup listening to send Google Analytics events when any element
 // with a 'data-ga-click-event' attribute is clicked.
 window.addEventListener('DOMContentLoaded', function () {
   function sendEvent(e) {
-    var elem = e.currentTarget;
-
-    var data = {
-      'event_category': 'click',
-      'event_label': 'path:' + window.location.pathname,
-      'value': 1
-    };
-
-    // Events on links should be sent via beacon, see:
-    // https://support.google.com/analytics/answer/7478520?hl=en
-    if (elem.hasAttribute('href')) {
-      var updated = false;
-      var callbackFn = function () {
-        if (updated) return;
-        updated = true;
-        document.location = elem.href;
-      };
-      data.event_callback = callbackFn;
-      data.transport_type = 'beacon';
```

Comment:
Removing this will break the tracking on links, wouldn't it?

Reply:
Yeah, I think it might... but that doesn't work anymore regardless..

Reply:
yes